docs/sections/Docs-Panels/index.jsx

```diff
@@ -3,8 +3,96 @@ import React from 'react';
 export default class Panels extends React.Component {
   render() {
     return (
-      <div>
-      </div>
+    	<div>
+    		<h2>Basic Panel</h2>
+    		<div className="panel -default">
+    			<div className="panel-body">Basic Panel Example</div>
+    		</div>
+
+    		<h2>Panel with Header and Body</h2>
+	      	<div className="panel -default">
+	      		<div className="panel-header">Panel Title</div>
```

Comment:
Again, with the conventions.

Let's prefer `.heading` over `.panel-heading` because it seems redundant. Check [rscss](rscss.io)